components/raft_log_engine/src/engine.rs

```diff
@@ -165,4 +165,8 @@ impl RaftEngine for RaftLogEngine {
     fn stop(&self) {
         self.0.stop();
     }
+
+    fn dump_stats(&self) -> Result<String> {
+        Ok("".to_owned())
```

Comment:
Should add a TODO comment if plan to support it in the future.

Reply:
I will leave a comment here.